rest/route/middleware.go

```diff
@@ -802,10 +801,12 @@ func AddCORSHeaders(allowedOrigins []string, next http.HandlerFunc) http.Handler
 		requester := r.Header.Get("Origin")
 		grip.Debug(message.Fields{
 			"op":              "addCORSHeaders",
+			"request":         r,
```

Comment:
What does this add for us?

Reply:
This was logging the entire request that was being sent i was hoping to use it to identify some request metadata that i may have missed but i can remove it.